### PR TITLE
Feat/ingress egress support for vpc sc

### PR DESCRIPTION
**Pull request by maxi-cit**
This adds support to ingress and egress rules within the perimeter in restricted vpc at 3-Network step, both on dual-svpc and hub-and-spoke

### CODE REVIEW

File: 3-networks-dual-svpc/envs/development/main.tf [hunk 1]
```diff
@@ -73,6 +73,8 @@ module "base_env" {
   default_region2                    = local.default_region2
   domain                             = var.domain
   parent_folder                      = var.parent_folder
+  ingress_policies                   = var.ingress_policies
+  egress_policies                    = var.egress_policies
```
Comment (bharathkkb) [inline 3-networks-dual-svpc/envs/development/main.tf:77]: Let's directly set them here as opposed to using a var.

Comment (daniel-cit) [reply 3-networks-dual-svpc/envs/development/main.tf:77]: Hi @bharathkkb  do you mean?
```
  ingress_policies                   = []
  egress_policies                    = []
```

Comment (maxi-cit) [reply 3-networks-dual-svpc/envs/development/main.tf:77]: Hi Barath, variables were added there to support some assertions on integrations tests. Should I still delete them?

Comment (bharathkkb) [reply 3-networks-dual-svpc/envs/development/main.tf:77]: Sounds good, lets keep for now. Looks like also need a rebase.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**